CMR/python/cmr/auth/token.py

```diff
@@ -56,48 +55,56 @@ def token_literal(token_text):
     return lambda _ : token_text
 
 
-def token_config(options: dict = None):
+# document-it: {"key":"cmr.token.value", "default":"None"}
+def token_config(config: dict = None) -> str:
     """
     Pull a token from the configuration dictionary
     Parameters:
-        options (dictionary): Responds to:
+        config: Responds to:
             "cmr.token.value": value of token, defaults to 'None'
     """
-    options = options if isinstance(options, dict) else {}
-    return options.get("cmr.token.value", None)
+    config = config if isinstance(config, dict) else {}
```

Comment:
This line is used six times in this pull request alone. Maybe modularize it?

Reply:
move to common